include/igl/find_cross_field_singularities.h

```diff
@@ -18,7 +18,7 @@ namespace igl
   // Inputs:
   //   V                #V by 3 eigen Matrix of mesh vertex 3D positions
   //   F                #F by 3 eigen Matrix of face (quad) indices
-  //   Handle_MMatch    #F by 3 eigen Matrix containing the integer missmatch of the cross field
+  //   Handle_MMatch    #F by 3 eigen Matrix containing the integer mismatch of the cross field
```

Comment:
This is sort of a nit, but above you renamed `Handle_MMatch` to `mismatch`. Can you do this again here so the names of variables are consistent across functions? Consistent naming makes it easier to figure out what's going on by looking at a function's signature.

Reply:
@fwilliams  Thanks, I overlooked this one, done.